### PR TITLE
Add Gitter Badge to README.md

### PR DESCRIPTION
**Pull request by SudharakaP**
I wanted to find the link to our public Gitter Channel recently and it took me a while to finally get to our [Community help](https://www.jhipster.tech/help/) page where it's located. 

Therefore I thought maybe we should add a Gitter badge to our README.md so that users could easily find it. Feel free to suggest an alternative place(s) to add this link. :smile: 

-   Please make sure the below checklist is followed for Pull Requests.

-   [x] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
-   [x] Tests are added where necessary
-   [x] Documentation is added/updated where necessary
-   [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 [![Logo][jhipster-image]][jhipster-url]
 
-[![NPM version][npm-image]][npm-url] [![Dependency Status][daviddm-image]][daviddm-url] [![Downloads][npmcharts-image]][npmcharts-url]
+[![NPM version][npm-image]][npm-url] [![Dependency Status][daviddm-image]][daviddm-url] [![Downloads][npmcharts-image]][npmcharts-url] [![Gitter](https://badges.gitter.im/jhipster/generator-jhipster.svg)](https://gitter.im/jhipster/generator-jhipster?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
```
Comment (MathieuAA) [inline README.md]: Can you do the same as previously done with the URL? I mean:
  - like that `[![NPM version][npm-image]][npm-url]`

See the bottom of the file to find the "aliases"

File: README.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
...
+
```
Comment (MathieuAA) [inline README.md]: there seems to be an extra line there, isn't there?

File: README.md (hunk 1 above)
Comment (SudharakaP) [reply README.md]: Ops, just copied and pasted without checking. :smile: Done.

File: README.md (hunk 2 above)
Comment (SudharakaP) [reply README.md]: Yep, my mistake. Thanks. :smile:

**Comment by MathieuAA** [comment]
Thanks!

**Comment by SudharakaP** [comment]
@MathieuAA : You are welcome; and thanks for the reviews. :+1:

**Pull request merged**